tests/system/robot/chromeTests.py

```diff
@@ -58,49 +58,181 @@ def checkbox_labelled_by_inner_element():
+		message="Browse mode: Move by word to word with details",
+	)
+	_asserts.braille_matches(
+		actualBraille,
+		"mln edt The word  hlght details cat hlght end  has a comment tied to it. edt end",
+		message="Browse mode: Move by word to word with details",
 	)
 	# read the details summary
-	actualSpeech = _chrome.getSpeechAfterKey("NVDA+\\")
-	_asserts.strings_match(
+	actualSpeech, actualBraille = _NvdaLib.getSpeechAndBrailleAfterKey(READ_DETAILS_GESTURE)
+	_asserts.speech_matches(
+		actualSpeech,
+		"Cats go woof BTW  Jonathon Commentor No they don't  Zara",
+		message="Browse mode: Report details on word with details"
+	)
+	_asserts.braille_matches(
+		actualBraille,
+		"Cats go woof BTW\n—Jonathon CommentorNo they don't\n—Zara",
+		message="Browse mode: Report details on word with details",
+	)
+
+	# Reset caret
+	actualSpeech = _NvdaLib.getSpeechAfterKey("upArrow")
+	_asserts.speech_matches(
 		actualSpeech,
-		"Cats go woof BTW  Jonathon Commentor No they don't  Zara Submit"
+		SPEECH_SEP.join([
+			"out of edit",
+			"Test page load complete",
+		]),
+		message="reset caret",
+	)
+
+	# Force focus mode
+	actualSpeech = _NvdaLib.getSpeechAfterKey("NVDA+space")
+	_asserts.speech_matches(
+		actualSpeech,
+		"Focus mode",
+		message="force focus mode",
+	)
+
+	# Tab into the contenteditable
+	actualSpeech, actualBraille = _NvdaLib.getSpeechAndBrailleAfterKey("tab")
+	_asserts.speech_matches(
+		actualSpeech,
+		SPEECH_SEP.join([
+			"edit",
+			"multi line",
+			"The word",  # content
+			"highlighted",
+			"has details",
+			"cat",  # highlighted content
+			"out of highlighted",
+			"has a comment tied to it.",  # content
+		]),
+		message="Focus mode: report content editable with details"
+	)
+	_asserts.braille_matches(
+		actualBraille,
+		"The word  hlght details cat hlght end  has a comment tied to it.",
+		message="Focus mode: report content editable with details",
+	)
+
+	# Try to read the details
+	actualSpeech, actualBraille = _NvdaLib.getSpeechAndBrailleAfterKey(READ_DETAILS_GESTURE)
+	_asserts.speech_matches(
+		actualSpeech,
+		SPEECH_SEP.join([
+			"No additional details",
+		]),
+		message="Focus mode: Try to read details, caret not on details word.",
+	)
+	_asserts.braille_matches(
+		actualBraille,
+		# a backslash character is typed, gesture only mapped in browse mode
+		"No additional details",
+		message="Focus mode: Try to read details, caret not on details word.",
+	)
+
+	# move to the word with details: "cat"
+	_NvdaLib.getSpeechAfterKey("control+rightArrow")
+	actualSpeech, actualBraille = _NvdaLib.getSpeechAndBrailleAfterKey("control+rightArrow")
+	_asserts.speech_matches(
+		actualSpeech,
+		SPEECH_SEP.join([
+			"highlighted",
+			"has details",
+			"cat",  # highlighted content
+			"out of highlighted",
+		]),
+		message="Focus mode: Move by word to word with details"
+	)
+	_asserts.braille_matches(
+		actualBraille,
+		# a backslash character was typed when the gesture was used

```

Comment:
The backslashes seem to be stripped now, I'm guessing this comment is no longer necessary?
Has the behaviour changed, and this no longer happens in NVDA?

Reply:
I missed the comment. If you look at the history of the file, in the earlier commits the backslash was sent to the application because the gesture was only in browse mode.

Reply:
Oh right, that makes sense. There's a few of these still in the document.